lib/dal/src/func/summary.rs

```diff
@@ -32,49 +32,35 @@ pub struct FuncSummary {
 }
 
 impl FuncSummary {
-    /// Returns the [summaries](FuncSummary) for all [`Funcs`](Func) in the [`ChangeSet`](crate::ChangeSet).
+    /// Returns the [summaries](FuncSummary) for all [`Funcs`](Func) in the workspace.
+    ///
+    /// This includes [`Funcs`](Func) that are "free floating" and not used by any [`Schema`] at present.
     pub async fn list(ctx: &DalContext) -> FuncSummaryResult<Vec<Self>> {
-        Self::list_inner(ctx, None).await
+        let funcs = Func::list(ctx).await?;
+        Ok(Self::from_funcs(funcs.as_slice()))
```

Comment:
Now, we collect all funcs in the workspace in the frontend rather than those are used by at least one schema.